website/www/site/content/en/documentation/io/built-in/snowflake.md

```diff
@@ -362,3 +837,209 @@ static SnowflakeIO.CsvMapper<GenericRecord> getCsvMapper() {
            };
 }
 {{< /highlight >}}
+## Cross Language
+### Intro
+Cross-language which is part of [Portability Framework Roadmap](https://beam.apache.org/roadmap/portability/) which aims to provide full interoperability across the Beam ecosystem. From a developer perspective it means the possibility of combining transforms written in different languages(Java/Python/Go). 
+
+Snowflake cross-language implementation is supporting both reading and writing operations for Python programming language. 
+
+Currently, cross-language is supporting only [Apache Flink](https://flink.apache.org/) as a runner in a stable manner but plans are to support all runners. For more information about cross-language please see [multi sdk efforts](https://beam.apache.org/roadmap/connectors-multi-sdk/) and [Beam on top of Flink](https://flink.apache.org/ecosystem/2020/02/22/apache-beam-how-beam-runs-on-top-of-flink.html) articles. 
+
+### Set up 
+Please see [Apache Beam with Flink runner](https://beam.apache.org/documentation/runners/flink/) for a setup. The specific setup for current version of snowflake is following:
+
+1. Setup a Flink cluster by following the Flink [Setup Quickstart](https://ci.apache.org/projects/flink/flink-docs-release-1.10/getting-started/tutorials/local_setup.html) or [Setting up Apache Flink on Mac OS X](https://streambench.wordpress.com/2017/10/26/setting-up-apache-flink-on-mac-os-x/)
+2. Download Job server image:
+`docker pull gcr.io/snowflake-poli/apachebeam_flink1.10_job_server:snowflake`
+3. Download Apache Beam Java SDK image:
+`docker pull gcr.io/snowflake-poli/apachebeam_java_sdk:2.20.0.dev`
+4. Change tag of downloaded Java SDK image to make the whole setup work:
+{{< highlight >}}
+docker tag gcr.io/snowflake-poli/apachebeam_java_sdk:2.20.0.dev apache/beam_java_sdk:2.20.0.dev
+{{< /highlight >}}
+5. Start Job server:
+{{< highlight >}}
+docker run -p 8099:8099 -p 8098:8098 -p 8097:8097 gcr.io/snowflake-poli/apachebeam_flink1.10_job_server:snowflake
+{{< /highlight >}}
+6. Download [Apache Beam Python SDK](https://storage.cloud.google.com/snowflake_artifacts/apache_beam-2.21.0-py2-none-any.whl?_ga=2.24638847.-471657054.1583857613).
+7. Install python Apache Beam Python SDK using python 2.7:
+`python -m pip install apache_beam-2.21.0-py2-none-any.whl`
+8. Run your job:
+`python script.py`
+### Reading from Snowflake 
+One of the functions of SnowflakeIO is reading Snowflake tables - either full tables via table name or custom data via query. Output of the read transform is a [PCollection](https://beam.apache.org/releases/pydoc/2.20.0/apache_beam.pvalue.html#apache_beam.pvalue.PCollection) of user-defined data type.
```

Comment:
The link is pointing to beam version 2.20.0

Reply:
fixed, now it is pointing to 2.23